types/index.d.ts

```diff
@@ -438,7 +438,12 @@ declare module "miragejs/orm/schema" {
   /**
    * An interface to the Mirage ORM that allows for querying and creating records.
    */
-  export default class Schema<Registry extends Record<string, ModelInstance> = Record<string, ModelInstance>> {
+  export default class Schema<
+    Registry extends Record<string, ModelInstance> = Record<
+      string,
+      ModelInstance
+    >
```

Comment:
Instead of defining `Registry` here, we should be able to 
```ts
import { Registry } from "miragejs";
```
and here
```ts
  export default class Schema<R extends Registry> {
    ⋮
    create<
      K extends keyof R,
      Init extends R[K],
      Data extends Partial<ModelInitializer<Init>>
    >(
      modelName: K,
      data?: Data
    ): Init & { [K in keyof Init & keyof Data]: Exclude<Init[K], undefined> };
    ⋮
```
I guess this could be a later refinement, though? v0v

Reply:
I updated the whole Schema definition based on your suggestion. Really appreciate. :)